src/components/Table/TablePropTypes.js

```diff
@@ -112,6 +112,7 @@ export const TableColumnsPropTypes = PropTypes.arrayOf(
     filter: PropTypes.shape({
       /** I18N text for the filter */
       placeholderText: PropTypes.string,
+      isMultiselect: PropTypes.bool,
```

Comment:
Could you add the same prop comments as above?
```suggestion
/** if isMultiselect and isFilterable are true, the table is filtered based on a multiselect */
      isMultiselect: PropTypes.bool,
```

Reply:
Added